src/Material/Tooltip.elm

```diff
@@ -0,0 +1,523 @@
+module Material.Tooltip
+  exposing
+    ( Model
+    , defaultModel
+    , Msg(..)
+    , update
+    , view
+    , render
+    , DOMState
+    , left
+    , right
+    , top
+    , bottom
+    , default
+    , large
+    , Property
+    , onMouseEnter

```

Comment:
I don't think we should expose `onMouseEnter` / `onMouseLeave`; user's should always just use `attach`, yes?